scripts/west_commands/sign.py

```diff
@@ -17,11 +17,12 @@
     FIND_BUILD_DIR_DESCRIPTION
 from runners.core import BuildConfiguration
 from zcmake import CMakeCache
-from zephyr_ext_common import Forceable, cached_runner_config
+from zephyr_ext_common import Forceable, cached_runner_config, \
+    zephyr_scripts_path
+
+scripts = zephyr_scripts_path()
```

Comment:
Getting rid of this variable will make it clear that zephyr_scripts_path() isn't used elsewhere.

Reply:
That's not a Zephyr style guide requirement, but sure, I will make that change.